src/Conf.ml

```diff
@@ -50,7 +50,8 @@ type t =
   ; space_around_collection_expressions: bool
   ; type_decl: [`Compact | `Sparse]
   ; wrap_comments: bool
-  ; wrap_fun_args: bool }
+  ; wrap_fun_args: bool
+  ; end_of_line_in: bool }
```

Comment:
We would like to keep the options alphabetically sorted in Conf.ml and Conf.mli (in type definitions, profiles, etc.)